opslevel/resource_opslevel_service.go

```diff
@@ -161,6 +163,11 @@ func (r *ServiceResource) Schema(ctx context.Context, req resource.SchemaRequest
 				Description: "The team that owns the service. ID or Alias may be used.",
 				Optional:    true,
 			},
+			"parent": schema.StringAttribute{
+				Description: "The id of the parent system of this service",
+				Optional:    true,
+				Validators:  []validator.String{IdStringValidator()},
```

Comment:
@davidbloss should this validator be here for `parent`? It looks like aliases can be used here just like in `owner`.

Reply:
Great catch 👏  - yes aliases should work too. Updating incoming